htdocs/core/class/commonobject.class.php

```diff
@@ -6306,6 +6306,8 @@ function showOptionals($extrafields, $mode='view', $params=null, $keysuffix='',
 			$out .= '<!-- showOptionalsInput --> ';
 			$out .= "\n";
 
+			$out .= '<table id="tableoptionals" class="" width="100%">';
```

Comment:
The method showOptionals is called in common case, inside a closing tr and opening tr.
So opening a table here is forbidden.
May be you can open this on the caller.